mask.json

```diff
@@ -1,14 +1,15 @@
 {
     "id": "mask",
-    "summary": "Apply a mask",
-    "description": "Applies a mask to a raster data cube. A mask can either be specified as:\n\n* **Raster data cube** for which parallel pixels among `data` and `mask` are compared and those pixels in `data` are replaced, which are non-zero (for numbers) or `true` (for boolean values) in `mask`.\n* **GeoJSON or vector data cube** containing one or more polygons. All pixels for which the point at the pixel center intersects with the corresponding polygon (as defined in the Simple Features standard by the OGC) are replaced.\n\nThe pixel values are replaced with the value specified for `replacement`, which defaults to `null` (no data). No data values will be left untouched by the masking operation.",
+    "summary": "Apply a raster mask",
+    "description": "Applies a mask to a raster data cube. A mask here is a raster data cube for which parallel pixels among `data` and `mask` are compared and those pixels in `data` are replaced for which a condition in `mask` is fulfilled (see parameter `expression`).\n\nThe pixel values are replaced with the value specified for `replacement`, which defaults to `null` (no data). No data values in `data` or `mask` will be left untouched by the masking operation.",
```

Comment:
I'm more a fan of using "corresponding pixels" instead of "parallel pixels"

Apart from that: there should probably be a note about how this parallellism/correspondance is handled in terms of cube dimensions and shape. Simple approach: dimensions (and shape) should be the same. But in practice one probably wants also be able to mask (x, y, t, band) data cubes with (x,y) masks?

Reply:
Indeed, suggestions how to phrase the latter?

Reply:
Telco: Will be tackled in a separate PR.

Reply:
Oh, seems there's already a related issue: #55